Lib/test/test_ssl.py

```diff
@@ -4532,8 +4534,8 @@ def msg_cb(conn, direction, version, content_type, msg_type, data):
                 # test sometimes fails with EOF error. Test passes as long as
                 # server aborts connection with an error.
                 with self.assertRaisesRegex(
-                    ssl.SSLError,
-                    '(certificate required|EOF occurred)'
+                    (ssl.SSLError, OSError),
```

Comment:
`ssl.SSLError` is a subclass of `OSError`.
```suggestion
                    OSError,
```